AngularInventory/src/styles.css

```diff
@@ -22,9 +22,24 @@ h1 {
 }
 
 .card {
-    max-width: 250px;
+    width: 500px;
+    box-shadow: 5px 10px 18px #888888;
```

Comment:
Use sass to define colors as named variables, and use named variables for colors.